tests/specs/scoped.js

```diff
@@ -592,7 +592,7 @@ test('scoped Store .off()', function (t) {
   })
 })
 
-test('when type change', function (t) {
+test.only('when type change', function (t) {
```

Comment:
Fixed, thanks. And there I thought I wouldn’t need the LGTM workflow as it was "only" a little fix :) Thanks so much for looking into it nevertheless